cmd/commands/cli/command_orders.go

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (C) 2020 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package cli
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/mysteriumnetwork/node/tequilapi/contract"
+	"github.com/pkg/errors"
+)
+
+func (c *cliApp) order(argsString string) {
+	var usage = strings.Join([]string{
+		"Usage: order <action> [args]",
+		"Available actions:",
+		"  " + usageOrderCreate,
+		"  " + usageOrderGet,
+		"  " + usageOrderGetAll,
+	}, "\n")
+
+	if len(argsString) == 0 {
+		info(usage)
+		return
+	}
+
+	args := strings.Fields(argsString)
+	action := args[0]
+	actionArgs := args[1:]
+
+	switch action {
+	case "create":
+		c.orderCreate(actionArgs)
+	case "get":
+		c.orderGet(actionArgs)
+	case "get-all":
+		c.orderGetAll(actionArgs)
+	default:
+		warnf("Unknown sub-command '%s'\n", argsString)
+		fmt.Println(usage)
+	}
+}
+
+const usageOrderCreate = "create [identity] [amount] [price currency] [receive currency] [use lightning network]"
```

Comment:
what's receive currency? Is this the currency we will get paid in? If so, should this even be configurable, should it not be hardcoded in pilvytis instead?

Reply:
Agree, this should be configurable in Pilvytis (ETH), not in node. Node should not pass the receive currency.

Reply:
> Pilvitys API is subject to change which might affect this PR

At the time of writing this code, this was the API. Now some changes have been done in master where we no longer send the receive currency, so I'm gonna alter this a bit.